swift3/utils.py

```diff
@@ -228,3 +228,8 @@ def extract_s3_etag(content_type):
         else:
             content_type += ';%s=%s' % (key, value)
     return content_type, s3_etag
+
+
+def add_s3api_to_log(req, apiname):
```

Comment:
`log_s3api_command` would be clearer. And a bit of documentation would help to understand the purpose of this function.